### ISSUE TITLE
[FE] Text box for the email.

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#91]
We can input the mail into the frontend

## Acceptance criteria
- [ ] there is a box where we can input the mail
- [ ] the mail is correctly given to the Backend and  the ticket notification is send to that mail.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by AnnaH3003** [comment]
already 2 points
